test-helpers/src/main/java/com/okta/testhelpers/OktaRule.kt

```diff
@@ -54,11 +55,12 @@ class OktaRule(
     fun createConfiguration(
         okHttpClient: OkHttpClient = this.okHttpClient,
         cache: Cache = AuthFoundationDefaults.cache,
+        eventHandlers: List<EventHandler> = emptyList(),
     ) = OidcConfiguration(
         clientId = "unit_test_client_id",
         defaultScope = "openid email profile offline_access",
         okHttpClientFactory = { okHttpClient },
-        eventCoordinator = EventCoordinator(eventHandler),
+        eventCoordinator = EventCoordinator(listOf(eventHandler) + eventHandlers),
```

Comment:
You might be able to avoid a lot of the dance (here and in individual tests) by using the `RecordingEventHandler.registerEventHandler`

Reply:
Oh cool. Did not realize that existed